frontend/src/views/Game.tsx

```diff
@@ -60,6 +60,7 @@ function GamePage() {
       setSocketSubscribed(true);
 
       // Check if end game.
+      // TODO: after deconstructing game, move this to useEffect on timeUp
```

Comment:
The deconstructing part should be already implemented (either in master or one of the many open PRs haha) 

Sounds good tho for the useEffect part - since it's relatively easy (1-commit change) I just added it as part of the [end game when everyone submits story](https://trello.com/c/ilAQWiO8/225-after-everyone-submits-end-game-add-useeffect-variable) since I figure it groups nicely with that one - feel free to modify this tho

Reply:
That sounds good!